src/frame/mod.rs

```diff
@@ -52,13 +52,8 @@ pub use self::stream_id::StreamId;
 pub use self::window_update::WindowUpdate;
 
 // Re-export some constants
-pub use self::settings::{
-    DEFAULT_SETTINGS_HEADER_TABLE_SIZE,
-    DEFAULT_INITIAL_WINDOW_SIZE,
-    DEFAULT_MAX_FRAME_SIZE,
-    MAX_INITIAL_WINDOW_SIZE,
-    MAX_MAX_FRAME_SIZE,
-};
```

Comment:
I know its not that common, but I personally prefer the imports on separate lines like this. The same line way gets weird and uglier as you start needing new lines eventually, but then with odd alignment indenting... And diffs.

Reply:
as far as i can tell rustfmt doesn't offer a way to configure this at the moment. is this a blocker?

Reply:
oh, nope. found a setting